src/Cli/dotnet/commands/dotnet-workload/install/LocalizableStrings.resx

```diff
@@ -166,40 +166,40 @@
     <value>Workload installation failed: {0}</value>
   </data>
   <data name="WorkloadCacheDownloadFailed" xml:space="preserve">
-    <value>Downloading workload packs to offline cache failed: {0}</value>
+    <value>Download of workload packs to offline cache failed: {0}</value>
   </data>
   <data name="WorkloadAlreadyInstalled" xml:space="preserve">
     <value>Workload '{0}' is already installed.</value>
   </data>
   <data name="InvalidNuGetVersionRange" xml:space="preserve">
-    <value>Specified version '{0}' is not a valid NuGet version range.</value>
+    <value>Specified version '{0}' isn't a valid NuGet version range.</value>
   </data>
   <data name="WorkloadPathOptionName" xml:space="preserve">
     <value>PATH</value>
   </data>
   <data name="WorkloadPathOptionDescription" xml:space="preserve">
-    <value>The directory where the workload will be installed. The directory will be created if it does not exist.</value>
+    <value>The directory where the workload will be installed. The directory will be created if it doesn't exist.</value>
   </data>
   <data name="PrintDownloadLinkOnlyDescription" xml:space="preserve">
-    <value>Only print the list of links to download without downloading it.</value>
+    <value>Only print the list of links to download without downloading.</value>
   </data>
   <data name="DownloadToCacheOptionDescription" xml:space="preserve">
-    <value>Download packages needed to install a workload to a folder which can be used for offline installation.</value>
+    <value>Download packages needed to install a workload to a folder that can be used for offline installation.</value>
   </data>
   <data name="FromCacheOptionDescription" xml:space="preserve">
     <value>Complete the operation from cache (offline).</value>
   </data>
   <data name="TempDirOptionDescription" xml:space="preserve">
-    <value>Configure the temporary directory used for this command (must be secure).</value>
+    <value>Specify a directory to use temporarily for this command (must be secure).</value>
```

Comment:
This still feels unclear to me, maybe we should specify that we're downloading temporary assets to this directory @tdykstra?

Reply:
Which assets are downloaded to this directory?

Reply:
All of the nuget packages that are required for the workload. Those packages are also extracted to this directory.

Reply:
How about

> Specify a temporary directory for this command to download and extract NuGet packages (must be secure).

Reply:
That sounds good to me, thanks!